tests/run-make/libs-through-symlinks/Makefile

```diff
@@ -3,10 +3,16 @@ include ../tools.mk
 
 # ignore-windows
 
+ifeq ($(UNAME),AIX)
+LN_FLAGS := -sf
+else
+LN_FLAGS := -nsf
+endif
```

Comment:
Suggestion: can you leave a comment here basically reproducing what you said in PR description?

Reply:
Added the comment as suggested, thanks!